magefile.go

```diff
@@ -28,8 +28,9 @@ var Aliases = map[string]interface{}{
 	"lint": mage.Linter.All,
 }
 
+// Build will create the project binaries found in /build/binaries
 func Build() {
-	sh.Run("go", "build")
+	sh.Run("goreleaser", "build", "--rm-dist", "--skip-validate")
```

Comment:
Is it intended to create a command for creating snapshots, passing the `--snapshot` flag?

Reply:
yes, still working on the mage cmds , several flags will be added, including builds only for current GOOS and GOARCH, filter by those criteria, PR is still a draft